doc/source/release-notes/Percona-Server-5.7.18-16.rst

```diff
@@ -0,0 +1,103 @@
+.. rn:: 5.7.18-16
+
+==========================
+|Percona Server| 5.7.18-16
+==========================
+
+Percona is glad to announce the GA (Generally Available) release of |Percona
+Server| 5.7.18-16 on July 28, 2017 (Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-5.7.18-16/>`_
+and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.7.18
+<http://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-18.html>`_, including
+all the bug fixes in it, |Percona Server| 5.7.18-16 is the current GA release
+in the |Percona Server| 5.7 series. All of |Percona|'s software is open-source
+and free, all the details of the release can be found in the `5.7.18-16
+milestone at
+Launchpad <https://launchpad.net/percona-server/+milestone/5.7.18-16>`_
+
+New Feature
+===========
+
+.. Do we mention that x64 is the only supported platform.
+
+|Percona Server| is now available on Debian 9 (stretch). The support only covers
+the ``amd64`` architecture.
+
+|Percona Server| can now be built with support of SSL 1.1.
```

Comment:
assuming s/SSL/OpenSSL is a way to say substitute OpenSSL for SSL due to its similarity to vim's syntax.